templates/home/index.html.twig

```diff
@@ -1,7 +1,58 @@
 {% extends 'base.html.twig' %}
 
-{% block title %}{{ someVariable }}{% endblock %}
+{% block title %}Projektai{% endblock %}
 
 {% block body %}
-    <div class="nfq-academy">{{ someVariable }}</div>
+
+    <ul class="list-group m-4">
+        <li class="list-group-item list-group-item-info">Studentai</li>
+        {% for student in group %}
+
+            <li class="list-group-item">
+                <a href="{{ path('student', {'name':  student.student  , 'project' : student.project}) }}">{{ student.student }}</a>
+                (
+                <span class="badge">Mentorius</span> {{ student.mentors.0 }}
```

Comment:
Nors pavyzdiniuose duomenyse yra tik 1 mentorius. Gera praktika radus masyvą (angl. _array_) naudoti `{% for`.
Taip įpratus bus lengviau koreguoti kodą kitiems (ir vis tiek veiks :wink:)